llvm/test/Transforms/LoadStoreVectorizer/NVPTX/256-bit.ll

```diff
@@ -0,0 +1,728 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
```

Comment:
I'd rename the test to reflect that it's for 256-bit loads and stores. E.g. `load-store-256-bit.ll`